docs/src/model_components/biogeochemical/index.md

```diff
@@ -5,7 +5,7 @@ Biogeochemical (BGC) models can be used within the [Oceananigans biogeochemistry
 For details of the BGC models currently implemented please see the following pages.
 
 ## Oceananigans setup
-At the simplest level all that is required to setup a BGC model is to pass it to the Oceananigans model setup:
+At the simplest level all that is required to setup an existint OceanBioME BGC model is to pass it to the Oceananigans model setup:
```

Comment:
```suggestion
At the simplest level all that is required to setup an existing OceanBioME BGC model is to pass it to the Oceananigans model setup:
```

Reply:
I think this is fixed in the current version right? https://github.com/OceanBioME/OceanBioME.jl/blob/41be828536b5f670a003eaeabffa819c77620223/docs/src/model_components/biogeochemical/index.md?plain=1#L8

Reply:
If you say so, I believe it!